editor/components/block-list/style.scss

```diff
@@ -1,46 +1,47 @@
+.editor-block-list__layout {
+	// prevent collapsing margins on container
+	padding-top: .1px;
+	padding-bottom: .1px;
+
+	// make room in the main content column for the side UI
+	// the side UI uses negative margins to position itself so as to not affect the block width
+	@include break-small() {
+		padding-left: $block-side-ui-padding;
```

Comment:
>Also noting that `.editor-block-list__layout` is wrapping a nested container, so these styles should apply there as well.

Speaking of... this is probably where the regression I described is introduced.

Reply:
I fixed it without touching this rule. This is part of the refactor, and it should, for now, bleed into the columns block intentionally. 

I fixed the appender by making it _more like a block_, when in fact it's actually a button.